lib/bulbs-elements/store/store.js

```diff
@@ -1,6 +1,29 @@
 import cloneObject from 'clone';
 import iterateObject from 'bulbs-elements/util/iterate-object';
 
+export default class Store {
+  constructor (options) {
+    this.actions = wrapActions(options.schema, this);
+    this.state = collectInitialState(options.schema);
+    this.components = [];
+    if (false) {
```

Comment:
The logging is a little noisy and we haven't made a choice on how to control it.

Sent from my iPhone

> On Apr 5, 2016, at 7:32 PM, spra85 <notifications@github.com> wrote:
> 
> In lib/bulbs-elements/store/store.js:
> 
> > @@ -1,6 +1,29 @@
> >  import cloneObject from 'clone';
> >  import iterateObject from 'bulbs-elements/util/iterate-object';
> >  
> > +export default class Store {
> > +  constructor (options) {
> > +    this.actions = wrapActions(options.schema, this);
> > +    this.state = collectInitialState(options.schema);
> > +    this.components = [];
> > +    if (false) {
> ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>